CAIPs/caip-341.md

```diff
@@ -0,0 +1,125 @@
+---
+caip: 341
+title: Extension ID Target Type Specification
+author: [Joao Tavares] (@ffmcgee725)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/341
+status: Draft
+type: Standard
+created: 2024-12-12
+requires: 294
+---
+
+## Simple Summary
+
+CAIP-341 defines the `extensionId` type as a valid target type for establishing connections with browser extension wallets via the [CAIP-294] `wallet_announce` wallet discovery event.
```

Comment:
thoughts on replacing all the of `extensionId` with Extension ID instead since it's not an actual property name or value we want someone using in their actual code?

Reply:
resolved in [79df192](https://github.com/ChainAgnostic/CAIPs/pull/341/commits/79df1928de49751d3192cc94d053db1e1e231948)